kong/plugins/cors/schema.lua

```diff
@@ -1,7 +1,21 @@
+local re_match = ngx.re.match
+
+local check_regex = function(value)
+  if value and (#value > 1 or value[1] ~= "*") then
+    for _, origin in ipairs(value) do
+      local _, err = re_match("just a string to test", origin)
+      if err then
+        return false, "origin " .. origin .. " is not a valid regex"
```

Comment:
Mind adding single quotes around the user-provided origin that failed this check? Error messages in our codebase do that, or at least, should :)

BTW: cool trick!